methods.tex

```diff
@@ -12,16 +12,15 @@ \subsection{Timing observables}
 The time delay between the end of the last iteration and exiting the \texttt{for} loop is $t_{\text{end\_loop}} = t_{\text{L6}}+t_{\text{L7}}$.
 The time $t_{\text{opening\_trajectory}} = t_{\text{L2}}+t_{\text{L3}}$ measures the problem setup, which includes data structure initialization and opening of topology and trajectory files.
 $t_{\text{Communication\_{MPI}}} = t_{\text{L16}}$ is the time to gather (``reduce'') all data from all processor ranks to rank zero.
-The total time (for all frames) spent in \texttt{block\_rmsd()} is $t_{\text{RMSD}} = t_{\text{L1}} + ...+ t_{\text{L8}}$. 
-There are parts of the code in \texttt{block\_rmsd()} that are not covered by the detailed timing information of \tcomp and \tIO. 
-To measure the un-accounted time we define the ``overheads''.
-$t_{\text{Overhead1}}$ and $t_{\text{Overhead2}}$ are the overhead of the calculations and they should be ideally very small (see Table \ref{tab:notation} for the definition).  
-The total time to completion for a single process on $N$ cores is $t_{N}$, which is mathematically equivalent to
-$ t_{\text{RMSD}} + \tcomm \equiv t_{N}$.
+The total time (for all frames) spent in \texttt{block\_rmsd()} is $t_{\text{RMSD}} = \sum_{i=1}^{8}t_{\text{Li}}$. 
+There are parts of the code in \texttt{block\_rmsd()} that are not covered by the detailed timing information of \tcomp and \tIO.
+
+Unaccounted time is considered as ``overhead''. We define
+$t_{\text{Overhead1}}$ and $t_{\text{Overhead2}}$ as the overhead of the calculations. They should ideally be very small (see Table \ref{tab:notation} for the definition). Finally, the total time to completion of a single process, when utilizing $N$ cores for the execution of the overall experiment, is $t_{N}$, as a result $t_{\text{RMSD}} + \tcomm \equiv t_{N}$.
```

Comment:
I believe t_{N} is not a good name for time to completion. Because the N in t_N does not have to do anything with N which is the number of cores. Do you have any suggestion on what to use instead of t_N?

Reply:
I think the original idea was that this is used in `S = t_1/t_N` and similar, i.e., the total time to completion depends on the number of cores.

Reply:
So, are we keeping the suggested change or you prefer to do something different?